spark_auto_mapper/automappers/with_column_base.py

```diff
@@ -73,6 +77,19 @@ def get_column_spec(self, source_df: Optional[DataFrame]) -> Column:
                         source_df=source_df, current_column=None, parent_columns=None
                     )
                 )
+
+            if self.skip_if_columns_do_not_exist:
+                columns_to_check = f"b.{self.skip_if_columns_do_not_exist[0]}"  # TODO: handle more than one
```

Comment:
if the plan is to support more than one column then i would say that should be implemented now instead of leaving a TODO.  is there a use case for supplying more than one column?

Reply:
I don't have any scenarios for multiple columns. I'll remove TODO list.

Reply:
ok, thanks. Can you please update the code to reflect that it should be a single optional string and not a list of strings?

Reply:
yes, I have updated to Optional[str]

Reply:
thanks. i hate to be a stickler but it would be good to make everything (field name, docstring, any variables, etc) also reflect that it is a single value. for example change `skip_if_columns_do_not_exist` to `skip_if_column_does_not_exist`